frame/support/src/storage/unhashed.rs

```diff
@@ -154,6 +154,16 @@ pub fn clear_prefix(
 	MultiRemovalResults { maybe_cursor, backend: i, unique: i, loops: i }
 }
 
+/// Returns `true` if the storage contains any key, which starts with a certain prefix,
+/// and is longer than said prefix.
+/// This means that a key which equals the prefix will not be counted.
+pub fn contains_prefixed_key(prefix: &[u8]) -> bool {
```

Comment:
@cheme I hope this is fine now

Reply:
just for some context, @shawntabrizi approved the api and doc 😺

Reply:
yes, if you look above, you will see a bunch of APIs for acting on one key, like `clear_prefix`, which use the same term and parameter.